File: codeforces/31.drinks/drinks.py
# Author: José Rodolfo (jric2002)
n = int(input())
p = list(map(int, input().split(" ")))
sum = 0
juice_volume = 0
for position in range(0, n):
  sum += p[position]
juice_volume = (sum / n)
print("{0:.12f}".format(juice_volume))

File: codeforces/low_difficulty/800_points/31.drinks_[200B]/drinks.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int n, p, sum = 0;
  float juice_volume;
  cin >> n;
  for (unsigned short int i = 0; i < n; i++) {
    cin >> p;
    sum += p;
  }
  juice_volume = (float)(sum) / n;
  cout << juice_volume << endl;
  return 0;
}